src/pages/SignUp.vue

```diff
@@ -47,16 +48,17 @@ export default {
   methods: {
     signup: function() {
       // TODO: get API from config
-      axios.post('https://private-8bf72-advex.apiary-mock.com/users',
+      axios.post('http://localhost:5000/users',
```

Comment:
Why using port 5000 here

Reply:
It's the default port of Flask app. While we can make this configurable, it's not very urgent because by Beta will be putting FE and BE in the same container.